app/models/customer.py

```diff
@@ -0,0 +1,27 @@
+from flask import current_app
+from app import db
+from sqlalchemy.orm import relationship
+
+class Customer(db.Model):
+    __tablename__ = "customer"
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    postal_code = db.Column(db.Integer)
+    phone = db.Column(db.String)
+    registered_at = db.Column(db.DateTime)
+    videos_checked_out_count = db.Column(db.Integer) #, default=1)
+    video = relationship("Rental", back_populates="customer")
```

Comment:
You're not really using this relationship, and it doesn't make sense for an attribute Video to be a rental.

This might work better and would give every Customer object a `.rentals` attribute, which is a list of the customer's rentals.

```suggestion
    rentals = db.relationship('Rental', backref='customer', lazy=True)
```